src/main/java/com/autotune/analyzer/deployment/KruizeDeployment.java

```diff
@@ -419,7 +419,8 @@ private static KruizeObject getAutotuneObject(String autotuneObjectJsonStr) {
                     targetCluster,
                     sloInfo,
                     selectorInfo,
-                    objectReference
+                    objectReference,
+                    null
```

Comment:
Why is this being set to null? We need to read in an `optstring` which will set to empty string if the user has not set it

Reply:
Ah! I missed that. I placed it for null testing, updated the code to substitute what user has sent.